addons/website/tests/test_ui.py

```diff
@@ -110,7 +110,10 @@ def test_admin_tour_rte_translator(self):
 @odoo.tests.common.tagged('post_install', '-at_install')
 class TestUi(odoo.tests.HttpCase):
 
-    def test_01_restricted_editor(self):
+    def test_01_admin_tour_homepage(self):
```

Comment:
@JKE-be Not sure, should we avoid to rename the other test methods in standard? I guess it's ok but to be sure...

Also: do we also adapt test_themes to make the tour run for each theme?